installers/go-agent/release/go-agent.init

```diff
@@ -96,7 +96,7 @@ stop_go_agent() {
     check_proc
 
     if [ $? -eq 0 ]; then
-        killproc -p $PID_FILE /usr/share/${SERVICE_NAME}/agent.sh >/dev/null
+        killproc
```

Comment:
Would you not pass an executable to killproc? Am I missing something?

Reply:
`killproc` is not the standard one. It's just a function in this file.